RELEASE_PROCESS.md

```diff
@@ -0,0 +1,246 @@
+# Release Process
+
+- [Release Process](#release-process)
+  - [Major Release Procedure](#major-release-procedure)
+    - [Tagging Procedure](#tagging-procedure)
+  - [Patch Release Procedure](#patch-release-procedure)
+  - [Major Release Maintenance](#major-release-maintenance)
+  - [Stable Release Policy](#stable-release-policy)
+- [Old Release Process](#old-release-process)
+  - [Release Procedure](#release-procedure)
+    - [Checks and tests](#checks-and-tests)
+    - [Major and minor Release](#major-and-minor-release)
+      - [example of releasing `v8.0.0-rc0`](#example-of-releasing-v800-rc0)
+      - [example of releasing `v8.0.0`](#example-of-releasing-v800)
+      - [example of releasing `v8.0.1`](#example-of-releasing-v801)
+    - [backport release](#backport-release)
+      - [example of backport release `v7.0.5`](#example-of-backport-release-v705)
+    - [Test building artifacts](#test-building-artifacts)
+    - [Release notes](#release-notes)
+
+
+This document outlines the release process for Cosmos Hub (Gaia).
+
+Gaia follows [semantic versioning](https://semver.org), but with the following deviations to account for state-machine breaking changes: 
+- Changes that requires upgrade trough governance will result in an increase of the major version X (X.y.z). Note that most likely this entails state-machine breaking changes.
+- Changes that are state-machine breaking but do not require upgrade through governance (e.g., emergency upgrades) will result in an increase of the minor version Y (x.Y.z | x > 0).
+- Changes that are not state-machine breaking will result in an increase of the patch version Z (x.x.Z | x > 0).
```

Comment:
This is similar to how we were already using versioning, but now we have it as a guideline.